lib/xcpretty/parser.rb

```diff
@@ -217,6 +217,61 @@ module Matchers
     # @regex Captured groups
     WRITE_AUXILIARY_FILES = /^Write auxiliary files/
 
+    # @regex Captured groups
+    # $1 = directory_path
+    # $2 = target
+    CREATE_BUILD_DIRECTORY = /^CreateBuildDirectory\s(\\ |[^ ]+)\s\(in target: (.*)\)/
```

Comment:
1. Done.
2. Yeah, looks like it, 	
3. Not all matchers do it, see `PREPROCESS_MATCHER` and `PBXCP_MATCHER`. My rule of thumb is to capture the full path where there might be multiple files with the same name, and just the file name otherwise.